download_image.sh

```diff
@@ -3,7 +3,7 @@ set -uo pipefail
 
 case $1 in
     "raspbian_lite:latest")
-        url=https://downloads.raspberrypi.org/raspbian_lite_latest
+        url=https://downloads.raspberrypi.org/raspios_lite_armhf/images/raspios_lite_armhf-2023-05-03/2023-05-03-raspios-bullseye-armhf-lite.img.xz
```

Comment:
It seems there is a confusion here between _raspian_lite_ (Buster) and _raspios_lite_ (Bullseye):
_raspbian_lite:latest_ should **not** be made to point to a Bullseye image.

Reply:
@aceisace exactly. Raspbian_lite latest really is latest Buster. I'll close this PR, but feel free to reopen to discuss this.